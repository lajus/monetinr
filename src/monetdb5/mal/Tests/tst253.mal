# testing void bats

b:= bat.new(:oid,:str);
bat.insert(b,nil:oid,"hello");
bat.insert(b,nil:oid,"world");
io.print(b);

#materialize header
c:= bat.reverse(b);
io.print(c);


d:= algebra.markT(c);
io.print(d);

#perform header marking
f:= algebra.markH(b);
io.print(f);
