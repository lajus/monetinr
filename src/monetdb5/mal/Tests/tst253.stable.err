stderr of test 'tst253` in directory 'monetdb5/mal` itself:
