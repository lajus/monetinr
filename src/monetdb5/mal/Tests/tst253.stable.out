stdout of test 'tst253` in directory 'monetdb5/mal` itself:


# 11:28:42 >  
# 11:28:42 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/log" --set mapi_port=43122 --set sql_port=57578 --set monet_prompt= --trace "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --dbname=mTests_src_mal  tst253.mal </dev/null
# 11:28:42 >  



# Monet Database Server V4.99.19
# Copyright (c) 1993-2005, CWI. All rights reserved.
# Compiled for x86_64-redhat-linux-gnu/64bit with 64bit OIDs; dynamically linked.
# config:/ufs/niels/scratch/Linux-x86_64/build-m5/etc/MonetDB5.conf
# dbfarm:/ufs/niels/scratch/Linux-x86_64/build-m5/var/MonetDB5/dbfarm
# dbname:mTests_src_mal
# Visit http://monetdb.cwi.nl/ for further information.
## testing void bats
#b:= new(:void,:str);
#insert(b,"hello");
#insert(b,nil,"world");
#print(b);
#
##materialize header
#print(c);
#d:= mark(c);
#print(d);
#
##perform header marking
#f:= algebra.markH(b);
#io.print(f);
#
function user.main():void;
# testing void bats 
    b := bat.new(:oid,:str);
    bat.insert(b,nil:oid,"hello");
    bat.insert(b,nil:oid,"world");
    io.print(b);
#materialize header 
    c := bat.reverse(b);
    io.print(c);
    d := algebra.markT(c);
    io.print(d);
#perform header marking 
    f := algebra.markH(b);
    io.print(f);
end main;
#-------------------------#
# h	tmp_4		  # name
# void	str		  # type
#-------------------------#
[ nil,	  "hello"	  ]
[ nil,	  "world"	  ]
#-------------------------#
# t		tmp_4	  # name
# str		void	  # type
#-------------------------#
[ "hello",	  nil	  ]
[ "world",	  nil	  ]
#---------------------------------#
# h		tmp_6		  # name
# str		void		  # type
#-------------------------#
# h	t	  # name
# str		void	  # type
#-------------------------#
[ "hello",	  0@0	  ]
[ "world",	  1@0	  ]
#-------------------------#
# t	h		  # name
# void	str		  # type
#-------------------------#
[ 0@0,	  "hello"	  ]
[ 1@0,	  "world"	  ]

# 11:28:42 >  
# 11:28:42 >  Done.
# 11:28:42 >  

